draft-ppm-protocol.md

```diff
@@ -437,13 +456,17 @@ number generator. Each task has the following parameters associated with it:
   newest report in a batch. This defines the boundaries with which the batch
   interval of each collect request must be aligned. (See
   {{batch-parameter-validation}}.)
-* `protocol`: named parameter identifying the core PPM protocol, e.g., Prio or
-  Hits.
+* `public_param` and `verify_param`: The outputs of the VDAF setup algorithm,
+  referred to as the public parameter and verification parameter respectively.
+  The client uses the poblic parameter to split its input into input shares, and
+  the aggregators use the verification parameter to verify the input's validity.
+  The public parameter MUST be distributed to all clients and MAY be distributed
+  to the servers. The verification parameter MUST be distributed to all
+  aggregators and MUST NOT be revealed to the clients.
```

Comment:
It's quite hard to understand the role of the verification parameter without a description of the vdaf system

Reply:
Ack, I'll add a description of VDAFs in the overview for this PR.